spec/requests/faqs_api.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+ RSpec.describe "Faqs API", type: :request do
+    let!(:category) { create_list(:categories_api, 10) }
```

Comment:
This line is incorrect.  You need a separate factory bot to create faq records.  I don't think the test can run successfully because you won't have any faqs loaded.  In the future please run rspec to run all tests to make sure they all pass before submitting the pull request.